gui/package.json

```diff
@@ -32,7 +32,7 @@
     "react-eject": "react-scripts eject",
     "electron-build": "electron-builder",
     "release": "npm run react-build && electron-builder --publish=always",
-    "build": "npm run react-build && npm run electron-build",
```

Comment:
Should probably make a new command instead of overwriting an existing one

Reply:
That was already like that, do we want it changed?